src/Compilers/VisualBasic/Portable/LanguageVersion.vb

```diff
@@ -27,7 +27,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     LanguageVersion.VisualBasic12,
                     LanguageVersion.VisualBasic14,
                     LanguageVersion.VisualBasic15
-
                     Return True
```

Comment:
>Return True [](start = 20, length = 11)

Please keep the empty line above. Since this is the only change in the file, consider simply reverting the file.